numismatic/feeds/base.py

```diff
@@ -22,6 +42,10 @@ class Feed(abc.ABC):
     rest_client = attr.ib(default=None)
     websocket_client = attr.ib(default=None)
         
+    @staticmethod
+    def get_symbol(asset, currency):
```

Comment:
Question: Should this be a static method or in the utils module?

Reply:
Has to be static since overriden by subclasses